src/timeline-list/index.tsx

```diff
@@ -49,17 +49,25 @@ export interface TimelineListProps {
    * Should initially scroll to a specific time (relevant only for NOT "today" timelines)
    */
   initialTime?: TimelineProps['initialTime'];
+  /** 
+   * Props to pass the InfinitList
+   */
+  infiniteListProps?: {
+    disableRecycling?: boolean;
+  }
 }
 
 const TimelineList = (props: TimelineListProps) => {
-  const {timelineProps, events, renderItem, showNowIndicator, scrollToFirst, scrollToNow, initialTime} = props;
+  const {timelineProps, events, renderItem, showNowIndicator, scrollToFirst, scrollToNow, initialTime, infiniteListProps} = props;
+  const disableRecycling = infiniteListProps?.disableRecycling;
+  const shouldFixRTL = disableRecycling || constants.isAndroidRTL; // isHorizontal = true
```

Comment:
Same here, for the user It's not clear that the prop purpose is to fix RTL.

Reply:
The prop purpose is not fixing RTL issue, the fix for RTL issue was to disable recycling... but now not anymore...